server/camphoric/views.py

```diff
@@ -187,6 +187,26 @@ def __init__(self, user_message):
         self.user_message = user_message
 
 
+class EventList(APIView):
+    def get(self, request):
+        '''
+        Return an array of objects with the following keys:
+        - name: event name
+        - url: url to the registration form for the event
+        - open: whether registration is open for this event
+        '''
+        events = models.Event.objects.all()
+
+        def map_event(event):
+            return {
+                'name': event.name,
+                'url': f"/events/{event.id}/register",
```

Comment:
Rather than hard-coding the url, you can use django.urls.reverse: https://docs.djangoproject.com/en/4.0/ref/urlresolvers/#reverse

Reply:
Actually, never mind - that's not an API URL.